src/playback/internal/au3/au3player.cpp

```diff
@@ -57,7 +57,8 @@ void Au3Player::play()
 
     Au3Project& project = projectRef();
 
-    auto options = ProjectAudioIO::GetDefaultOptions(project, true /*newDefault*/);
+    const bool newDefault = true; //(mode == PlayMode::loopedPlay);
```

Comment:
I don't really understand the meaning of `newDefault` but can you confirm that setting it to `True` from previous `False` does not break anything down the function?

Reply:
actually, newDefault should have always been true if no loop play, like in au3

Reply:
why do we need `if (newDefault)` on line 114 then if it is always true?

Reply:
Okay got it, I did not dive deep into what the code is doing, and the name of the varialbe just resets my brain when I'm trying to understand it. 
So the idea is to add more logic to it later, and keep it as it is for now?

Reply:
yes